uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/waiting-approval-notification/components/waiting-approval-notification.component.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-env amd */
+
+define([
+  'common/lodash',
+  'leave-absences/waiting-approval-notification/modules/components'
+], function (_, components) {
+  components.component('waitingApprovalNotification', {
```

Comment:
but why can't you do everything in one component? Why do you need two nested components just to show a badge?